stories/data/article-doc.json

```diff
@@ -136,145 +136,6 @@
             }
           ]
         },
-        {
-          "type": "multi_graphic_figure_element",
```

Comment:
Why is this deleted? How is it related to dependencies update?

Reply:
Part of the image changes was to remove the multi_graphic_figure_element node, so this doc fails to parse.